packages/server/src/routes/sessions.js

```diff
@@ -17,8 +17,22 @@ const {
 const MAX_ACCESS_TOKEN_USES = 1;
 
 function validatePostLoginRedirectPath(url) {
```

Comment:
I guess another way to go about this would be with an allowlist for known routes?  That's just not very straightforward right now, because server and client codebases are separated.

Reply:
Yeah, I thought about that and didn't do it for that reason.